homeassistant/util/yaml/loader.py

```diff
@@ -88,6 +97,30 @@ def _load_secret_yaml(self, secret_dir: Path) -> dict[str, str]:
         return secrets
 
 
+class FastestSafeLoader(FastestAvailableSafeLoader):
+    """The fast C safe loader."""
```

Comment:
this is not perse C

Reply:
True, but since pyyaml actually calls it the C loader I didn't want to call it something else.

Reply:
I meant that `FastestAvailableSafeLoader` might actually be a reference to `yaml.SafeLoader` because the import failed.

Reply:
Ahh. That makes more sense.  I'll rename it